scripts/normalize-by-median.py

```diff
@@ -189,19 +188,19 @@ def normalize_by_median_and_check(input_filename, htable, single_output_file,
 
     total_acc = None
     discarded_acc = None
-
+    corrupt = None
     if single_output_file:
         if single_output_file is sys.stdout:
             output_name = '/dev/stdout'
         else:
             output_name = single_output_file.name
         outfp = single_output_file
-
     else:
         output_name = os.path.basename(input_filename) + '.keep'
         outfp = open(output_name, 'w')
 
-    with CatchIOErrors(input_filename, outfp, fail_save, htable, force, norm):
+    with CatchIOErrors(input_filename, outfp, fail_save, htable, force) as \
```

Comment:
re: corrupt: any insight on how to get the list of corrupt files out of CatchIOErrors without throwing in more args? 

Originally I made the corrupt_files an attribute of the normalizer object, which worked but seemed messy. 

I tried passing in a list that was created out of CIOE but it appears to just copy it, and I can't yield it out magically and CIOE doesn't let me give it attributes (i.e., yielding out "self" and doing "CIOE.corrupt_files" on the outside)